### DISCUSSION TITLE
Finding medication type and a question on insurance types

### DISCUSSION THREAD

**Discussion by Daphnaidan** [MIT-LCP/mimic-code#1642] [MIMIC-IV]
Hi, 

If I can't find a medication in the inputevents table nor in the concept map - do I need to assume that it does not exist at all (I'm looking for erythropoietin)?

Another question I would like to ask is on insurance types - in the admissions table I see that there are only Medicaid and Medicare and the rest are "other" - is this because the rest are private insurance or that I need to assume that the data was not collected for all the patients? If it's indeed due to this reason, is there another source for this information in the database?

Thank you!

**Comment by alistairewj** [comment]
*inputevents* does not have an exhaustive map of medications. Take a look at either *prescriptions* or *emar*.

The insurance data is mapped to those categories out of simplicity. We could make it more granular if there is an obvious way to do it and/or specific advantage in doing so. Interested to hear your (or anyone else's) thoughts there.

**Comment by Daphnaidan** [comment]
Thank you! It helped :)
About the insurance - I found it valuable as a way of evaluating socioeconomic status, but I'm not sure it's the right way to do it since there are probably a lot of insurance types, and the granularity of this by itself is not something I necessarily find helpful.

**Comment by alistairewj** [comment]
Got it, thanks! We are looking at other measures of SES and hope to release
them soon.

**Reply by marymlucas** [reply]
That's great to hear and would be very helpful to have.

**Reply by KevinKloiber** [reply]
I just found this answer in the search related to socioeconomic status and I wanted to ask whether there is any update on the SES release. I hope this does not sound pushy, I really appreciate your great work!
